### PR TITLE
Adding notes for upgrading stemcells in genesis

### PR DESCRIPTION
**Pull request by cweibel**
Attempting to add instructions for upgrading stemcells leveraging defaults in the kits, in the voice and styling of Mr Hunt

### CODE REVIEW

File: genesis.md [hunk 1]
```diff
@@ -481,3 +481,40 @@ same principle can be used to scale up other aspects of
...
+## Stemcell Version Upgrades
+
+Genesis provides a simple way for you to update stemcells for deployments based off of Genesis Kits.  The typical process is to upload a new stemcell to the BOSH Director and perform a `genesis deploy` on each deployment.
+
+In each kit there is a default for the os flavor and version:
+
+```
+params:
+  stemcell_os:      ubuntu-trusty
+  stemcell_version: latest
```
Comment (jhunt) [inline genesis.md]: This is turning into `3468.latest` for reasons of compiled stemcells.

File: genesis.md [hunk 2]
```diff
@@ -481,3 +481,40 @@ same principle can be used to scale up other aspects of
...
+```
+
+To upgrade the stemcell version for a deployment start by uploading a new stemcell to the BOSH Director.  Full instuctions for doing this are located in the [BOSH Runbook under Uploading a Stemcell](bosh.md#upload-a-stemcell).
+
```
Comment (jhunt) [inline genesis.md]: You will not be able to link to the `*.md` files;  Check https://runbooks.starkandwayne.com for the URI to use to skip to another runbook.

File: genesis.md [hunk 3]
```diff
@@ -481,3 +481,40 @@ same principle can be used to scale up other aspects of
...
+  stemcell_version: see_options_below
+```
+
+`stemcell_version` options:
+
+  * `latest` - Grabs the highest major.minor stemcell for the stemcell_os
+  * `3541.2` - Hard codes a this specific major.minor stemcell version
+  * `<major>.latest` - Grabs the highest minor stemcell for the listed major version, (ie `3451.latest`)
+
+Once the new stemcell is uploaded to the BOSH Director and any overrides to the version are provided to the environment yaml file perform a `genesis deploy` on the kit deployment.
```
Comment (jhunt) [inline genesis.md]: This is a good point at which to point out that most kits ship pre-compiled BOSH releases and do not allow the major version of OS of the stemcell to be changed.  BOSH is this way.  So is CF.

File: genesis.md (hunk 1 above)
Comment (cweibel) [reply genesis.md]: Updated to `3468.latest`

File: genesis.md (hunk 2 above)
Comment (cweibel) [reply genesis.md]: Updated to use full url

File: genesis.md [hunk 4]
```diff
@@ -481,3 +481,43 @@ same principle can be used to scale up other aspects of
...
+
+In each kit there is a default for the os flavor and version:
+
+```
+params:
+  stemcell_os:      ubuntu-trusty
+  stemcell_version: 3468.latest
+```
+
+To upgrade the stemcell version for a deployment start by uploading a new stemcell to the BOSH Director.  Full instuctions for doing this are located in the [BOSH Runbook under Uploading a Stemcell](https://runbooks.starkandwayne.com/bosh.html#upload-a-stemcell).  It should be noted that most kits ship pre-compiled BOSH releases and do not allow the major version of OS of the stemcell to be changed.  If you are currently using `3468.22` you should upload a newer version in the `3468` series, such as `3468.23`.
```
Comment (jhunt) [inline genesis.md]: Using a path-only URL allows us to rename runbooks.starkandwayne.com (should we choose to do so).

```suggestion
To upgrade the stemcell version for a deployment start by uploading a new stemcell to the BOSH Director.  Full instuctions for doing this are located in the [BOSH Runbook under Uploading a Stemcell](/bosh.html#upload-a-stemcell).  It should be noted that most kits ship pre-compiled BOSH releases and do not allow the major version of OS of the stemcell to be changed.  If you are currently using `3468.22` you should upload a newer version in the `3468` series, such as `3468.23`.
```

**Pull request merged**